core/src/main/java/org/jdbi/v3/core/mapper/GetObjectColumnMapperFactory.java

```diff
@@ -61,12 +61,13 @@ public static ColumnMapperFactory forClasses(Set<Class<?>> types) {
     }
 
     @Override
+    @SuppressWarnings("unchecked")
     public Optional<ColumnMapper<?>> build(Type type, ConfigRegistry config) {
         return Optional.of(type)
             .filter(Class.class::isInstance)
             .map(Class.class::cast)
             .filter(supportedTypes::contains)
-            .map(GetObjectColumnMapper<Object>::new);
+            .map(GetObjectColumnMapper::new);
```

Comment:
This change looks like a step backward. We should avoid raw types if possible

Reply:
the compiler marked that as gray ("not needed").

Reply:
reverted